src/types/invites.rs

```diff
@@ -0,0 +1,27 @@
+#[derive(Serialize, Deserialize)]
+pub struct Invite {
+    /// The invite's unique code (string)
+    pub code: String,
+
+    /// The invite owner's user ID
+    ///
+    /// 128 bit unsigned integer
+    pub owner_id: u128,
+    
+    /// The invite's guild ID
+    ///
+    /// 128 bit unsigned integer
+    pub guild_id: u128,
+
+    /// When the invite was created as a timezone-naive 
```

Comment:
Did you mean
> timezone-naive datetime
```suggestion
    /// When the invite was created as a timezone-naive datetime
```

Reply:
yep i did, i was writing the line, and then forgot lol. lemme edit